backend/routes.py

```
@@ -35,17 +42,25 @@ def register_event_handlers():
     event_bus.register_handler(EventType.PR_COMMENTED, sample_handler, PlatformType.GITHUB)
     event_bus.register_handler(EventType.PR_MERGED, sample_handler, PlatformType.GITHUB)
 
+register_event_handlers()
+
 @router.post("/github/webhook")
 async def github_webhook(request: Request):
-    payload = await request.json()
+    try:
+        raw_payload = await request.json()
+        payload = GitHubWebhookPayload(**raw_payload)
+    except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)